### DISCUSSION TITLE
Not working on Ventura :(

### DISCUSSION THREAD

**Discussion by holmes123misc** [kyleneideck/BackgroundMusic#644] [Q&A]
i just upgraded and cant make Background music work. Does anyone have a tip?

**Comment by CarlSustrich** [comment]
Bummer, I came here just to see what to expect as I'm contemplating updating to Ventura. Did you end up getting this to work?

**Comment by teacherguy2020** [comment]
It works for me, but after installing I did need to restart the Music app. At first I got no sound from it

**Comment by NotTheVacuum** [comment]
I've tried a couple times and I can't get it to work. I've turned every slider but the main volume output to zero and every app still makes sound. The main volume slider works, but no per-app sliders do.

**Reply by icey0230** [reply]
me 2

**Comment by denisvvf** [comment]
Same here... not working. 
The apps show, but the volume control doesn't work.

**Comment by netscraper2** [comment]
Same here. Not working. :(
